exercises/practice/scrabble-score/.meta/example.awk

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env gawk -f
+
+BEGIN {
+    tile["A"]=1; tile["E"]=1; tile["I"]=1; tile["O"]=1; tile["U"]=1
+    tile["L"]=1; tile["N"]=1; tile["R"]=1; tile["S"]=1; tile["T"]=1
+    tile["D"]=2; tile["G"]=2
+    tile["B"]=3; tile["C"]=3; tile["M"]=3; tile["P"]=3
+    tile["F"]=4; tile["H"]=4; tile["V"]=4; tile["W"]=4; tile["Y"]=4
+    tile["K"]=5
+    tile["J"]=8; tile["X"]=8
+    tile["Q"]=10; tile["Z"]=10
+
+    # FS cannot contain space
+    FS = OFS = ","
+}
+
+{
+    $1 = toupper($1)
+    split($1, letters, //)
```

Comment:
Can't you also just do `FS = ""` to get fixed point?

Reply:
Sure can.

Reply:
And from reading about `FPAT` for ocr-numbers, can also do `FPAT="[[:alpha:]]"` -- the numbered fields are ***only the letters in the string***.
```
$ echo 'abc123def' | gawk -v FPAT='[[:alpha:]]' '{for (i=1; i<=NF; i++) print i, $i}'
1 a
2 b
3 c
4 d
5 e
6 f
```
How cool is that?

Reply:
Wow! That's incredibly useful. Apparently there's also `FIELDWIDTHS` for reading in blocks of bytes.